K2Bridge/Models/Response/Shards.cs

```diff
@@ -6,17 +6,38 @@ namespace K2Bridge.Models.Response
 {
     using Newtonsoft.Json;
 
+    /// <summary>
+    /// Object containing a count of shards used for the request.
+    /// </summary>
     public class Shards
     {
+        /// <summary>
+        /// Gets or sets the total number of shards.
+        /// It is here only to support the expected response.
+        /// And In our case it is a const == 1.
```

Comment:
Proposal: "Exists in order to support Kibana's expected response".

Also I would discard the word "And".

(Relevant for the following properties as well)